.github/workflows/wda-package.yml

```diff
@@ -2,53 +2,60 @@ name: Building WebDriverAgent
 
 on:
   workflow_dispatch:
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
```

Comment:
This kicks https://github.com/appium/WebDriverAgent/actions/workflows/wda-package.yml after each release. Still the upload destination is not GitHub releases https://github.com/appium/WebDriverAgent/releases, though. (Eventually the upload destination want to be the releases page)